.github/workflows/plugin-check.yml

```diff
@@ -0,0 +1,53 @@
+name: 'WordPress Plugin Checks'
+on: # rebuild any PRs and main branch changes
+  push:
+    branches:
+    - trunk
+    - develop
+
+  pull_request:
+    branches:
+      - trunk
+      - develop
+      - add/plugin-check
```

Comment:
Assuming we'll want to remove this prior to this being merged, so making a note here to remind us of that